src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsmanageprisonvisitsorchestration/config/ResourceServerConfiguration.kt

```diff
@@ -0,0 +1,38 @@
+package uk.gov.justice.digital.hmpps.hmppsmanageprisonvisitsorchestration.config
+
+import org.springframework.context.annotation.Bean
+import org.springframework.context.annotation.Configuration
+import org.springframework.security.config.annotation.method.configuration.EnableGlobalMethodSecurity
+import org.springframework.security.config.annotation.web.builders.HttpSecurity
+import org.springframework.security.config.annotation.web.configuration.EnableWebSecurity
+import org.springframework.security.config.http.SessionCreationPolicy
+import org.springframework.security.web.SecurityFilterChain
+
+@Configuration
+@EnableWebSecurity
+@EnableGlobalMethodSecurity(prePostEnabled = true, proxyTargetClass = true)
+class ResourceServerConfiguration {
+
+  @Bean
+  @Throws(Exception::class)
+  fun filterChain(http: HttpSecurity): SecurityFilterChain {
+    http
+      .sessionManagement()
+      .sessionCreationPolicy(SessionCreationPolicy.STATELESS)
+      .and().csrf().disable()
+      .authorizeRequests { auth ->
+        auth.antMatchers(
+          "/health/**", "/info",
+          "/v3/api-docs/**",
+          "/swagger-ui/**", "/swagger-ui.html",
+          "/swagger-resources",
+          "/swagger-resources/configuration/ui",
+          "/swagger-resources/configuration/security",
+          "/webjars/**", "/favicon.ico", "/csrf",
```

Comment:
whats web jars?

Reply:
Again a straight copy from VSIP. am not very sure about this bit.